components/utils/HttpRequest/src/main/java/org/mozilla/httprequest/HttpRequest.java

```diff
@@ -12,7 +12,7 @@
 public class HttpRequest {
 
     public static String get(URL url, final String userAgent) {
-        return get(url, -1, userAgent);
+        return get(url, 2000, userAgent);
```

Comment:
No magic numbers. Also, is there any reference prooving 2000 to be a good number? (My experiences says otherwise.)